src/backend/sideload/games.ts

```diff
@@ -197,6 +197,12 @@ export async function launchApp(appName: string): Promise<boolean> {
         }
       )
 
+      sendFrontendMessage('gameStatusUpdate', {
```

Comment:
one thing I'm confused here, shouldn't this change right BEFORE the callRunner call instead of AFTER?

because if `callRunner` is awaiting until you exit the game then the status will be `Launching` all the time until you exit the game, then it would change to `playing`

Reply:
I'll revert the commit that I added that changed the order